docs/content/en/docs-dev/user-guide/managing-piped/configuring-notifications.md

```diff
@@ -83,6 +83,23 @@ spec:
           mentionedAccounts:
             - '@user1'
             - 'user2'
+      - name: integration-slack-api-with-oauthTokenData-and-mentioned-groups
+        slack:
+          oauthTokenData: token
+          channelID: testid
+          mentionedGroups:
+            - 'group1'
+            - '@group2'
+      - name: integration-slack-api-with-oauthTokenData-and-mentioned-both-accounts-and-groups
+        slack:
+          oauthTokenData: token
+          channelID: testid
+          mentionedAccounts:
+            - 'user1'
+            - '@user2'
+          mentionedGroups:
+            - 'group1'
+            - '@group2'
```

Comment:
IMO, we should not use "@" to mention a slack group because the `handle` field after '@' (eg. `group2`) is mutable, it is not convenient since changing the handle involves updating configuration.
refs: https://slack.com/help/articles/212906697-Create-a-user-group

Thus, I suggest using `<!subteam^ID>` for group mentioning, since the fields are immutable and can be used according to [Slack's official docs](https://api.slack.com/reference/surfaces/formatting#mentioning-groups).

Reply:
@minhquang4334  thank you very much for your support, yes I agreed with your point
so should be support both `<!subteam^ID>` or only `ID`

Reply:
@minhquang4334 just from the time being, as slack's client, using slack to mention user and group has exactly same way (by `@`)

Reply:
thank you @hungran

Reply:
@hungran @minhquang4334 
As far as I know, we can only use immutable ID for the `<!subteam^ID>` format when we post messages by API.
If we want to provide a way to use the handle for group mentioning, we can use the web API `usergroups.list` method to retrieve the ID.
We can use group mentions only with paid workspace, so I'll verify the current implementation of this PR with my Slack workspace, which I use to work.
I must gain permission to register for the Slack application, so the verification takes time.

Reply:
thank you Warashi, yes we are following immutable ID `<!subteam^ID>`

Reply:
Oops, thank you.
Then, please change `groupname` to `groupID` in this document?
- `docs/content/en/docs-dev/user-guide/managing-piped/configuration-reference.md`

Reply:
I checked the behavior with this configuration.
```
spec:
  notification:
    mentions:
      - event: DEPLOYMENT_TRIGGERED
        slackGroups:
          - <!subteam^S07A5J95MHR>
          - S07A5J95MHR
          - <!subteam^warashi-test>
          - warashi-test
```

And I got this notification.
(I set the keyword notification, so please ignore `warashi` and `sawada` is highlighted)

<img width="587" alt="image" src="https://github.com/pipe-cd/pipecd/assets/3600530/2be60edc-29b6-4d47-9ebe-04dba1f0268d">

There are extra `<@` and `>` around the mention, and configuration only with the ID is not recognized as a mention.

Reply:
wired, they should be in `Mention To Group` let me double check

Reply:
Hi @Warashi a bit hard from my site to reproduce this error since i didnt have Slack workspace ent
@Warashi @khanhtc1202 @minhquang4334 could you help a look